onnxruntime/core/providers/cuda/nn/layer_norm_impl.cu

```diff
@@ -470,6 +470,7 @@ LAYERNORM_LINEAR_IMPL(float, float, float, false)
 LAYERNORM_LINEAR_IMPL(half, float, half, false)
 LAYERNORM_LINEAR_IMPL(double, double, double, false)
 LAYERNORM_LINEAR_IMPL(double, float, double, false)
+LAYERNORM_LINEAR_IMPL(double, float, double, true)
```

Comment:
It is possible to reuse `LAYERNORM_LINEAR_IMPL(double, double, double, true)`. See the other comment.